torch/_inductor/fx_passes/mkldnn_fusion.py

```diff
@@ -930,6 +930,14 @@ def _is_packable_linear(match):
         """
         Check if the node is supported for MKLDNN linear.
         """
+
+        def is_constant_weight(weight):
```

Comment:
Can you revise the naming of the function? It does not only check the weight is constant, right?

Reply:
Yes, thanks. Revised to is_const_or_cat_by_const